pkg/vtpm/src/server.cpp

```diff
@@ -18,6 +23,9 @@ using namespace google::protobuf::io;
 #define CODED_STRM_HDR_LEN 4
 #define SERVER_PORT 8877
 #define LISTEN_BACKLOG_LIMIT 16
+#define MAX_ARGS 15
+#define BIN_PATH "/usr/bin/"
+#define CMD_OUTPUT_FILE "cmd.output"
```

Comment:
If you have two commands at the same time, they would clobber the same file.
Does the TCP server ensure that you only run one command at a time?
If not you'd need a unique tmpfile

Reply:
Right now TCP server is blocking, so client requests are serialised.

Reply:
OK, it would be good to add a comment about that. I suspect that when we have multiple domUs using this we might want to avoid serialization even the TPM itself might serialize some of the work.